invenio_files_rest/__init__.py

```diff
@@ -6,7 +6,819 @@
+
+.. code-block:: json
+
+    {
+       "updated":"2019-05-17T07:07:22.219002+00:00",
+       "links":{
+          "self":"http://localhost:5000/files/
+                  c896d17b-0e7d-44b3-beba-7e43b0b1a7a4/my_file.txt?
+                  uploadId=a85b1cbd-4080-4c81-a95c-b4df5d1b615f",
+
+          "object":"http://localhost:5000/files/
+                    c896d17b-0e7d-44b3-beba-7e43b0b1a7a4/my_file.txt",
+
+          "bucket":"http://localhost:5000/files/
+                    c896d17b-0e7d-44b3-beba-7e43b0b1a7a4"
+       },
+       "last_part_size":5242880,
+       "created":"2019-05-17T07:07:22.218998+00:00",
+       "bucket":"c896d17b-0e7d-44b3-beba-7e43b0b1a7a4",
+       "completed":false,
+       "part_size":6291456,
+       "key":"my_file.txt",
+       "last_part_number":1,
+       "id":"a85b1cbd-4080-4c81-a95c-b4df5d1b615f",
+       "size":11534336
+    }
+
+Now, let's upload each part in parallel. Notice the :code:`uploadId` and
+:code:`partNumber` URL query arguments:
+
+.. code-block:: console
+
+   $ U=a85b1cbd-4080-4c81-a95c-b4df5d1b615f
+
+   $ curl -i -X PUT --data-binary @segment_aa \
+     "http://localhost:5000/files/$B/my_file.txt?uploadId=$U&partNumber=0"
+
+   $ curl -i -X PUT --data-binary @segment_ab \
+     "http://localhost:5000/files/$B/my_file.txt?uploadId=$U&partNumber=1"
+
+Complete the multipart upload:
+
+.. code-block:: console
+
+   $ curl -i -X POST \
+     "http://localhost:5000/files/$B/my_file.txt?uploadId=$U"
+
+You can also abort a multipart upload (and delete all uploaded parts):
+
+.. code-block:: console
+
+   $ curl -i -X DELETE "http://localhost:5000/files/$B/my_file.txt?uploadId=$U"
+
+
+.. _usage-large-files:
+
+Large Files
+-----------
+By default, Flask and your web server have a limit on the maximum size of the
+upload files. Normally, when the max size is exceeded, the server will return
+a response code :code:`413 (Request Entity Too Large)`.
+
+You can adjust these configurations according to your needs.
+
+For Flask, specify :code:`MAX_CONTENT_LENGTH` header. Be aware that if
+the request does not specify a :code:`CONTENT_LENGTH`, no data will be read.
+To change the max size, you can for example:
+
+.. code-block:: console
+
+    $ app.config['MAX_CONTENT_LENGTH'] = 25 * 1024 * 1024
+
+Here an example for Nginx web server. If you are using another web server,
+please check the related documentation.
+
+.. code-block:: console
+
+    http {
+        ...
+        client_max_body_size 25M;
+    }
+
+
+.. _usage-data-migration:
+
+Data Migration
+--------------
+When you already have an instance running with a certain amount of uploaded
+data, you might have the need to migrate the data to a different, larger or
+more efficient physical location. It can involve your entire set of files or
+just a part of it.
+
+Note that files migration can be performed with no downtime and in a
+completely transparent way for the user.
+
+The steps to follow to perform a complete migration are the followings:
```

Comment:
```suggestion
The steps to perform a complete migration are the following:
```
To avoid `follow*` repetition.